### ISSUE TITLE
Wrong version number in package

### ISSUE THREAD

**Issue by wilkovanderveen** [bartverdonck/Sitecore-Forms-Extensions#6]
When I download the 1.4.3. version of this extension, the package installer says 1.4.1

**Comment by bartverdonck** [comment]
Good point, sorry for that. We will make sure the version number is correct in the next release.
